magicparse/validators.py

```diff
@@ -36,4 +37,18 @@ def key() -> str:
         return "regex-matches"
 
 
-builtins = [RegexMatches]
+class GreaterThan(Validator):
+    def __init__(self, threshold: str) -> None:
+        self.threshold = Decimal(threshold)
+
+    def apply(self, value: Decimal) -> Decimal:
+        if value > self.threshold:
+            return value
+        raise ValueError(f"value is not greater than {self.threshold}")
```

Comment:
Rien de bloquant, mais de manière générale il vaut mieux préférer la forme positive dans les messages d'erreurs. Ca les rend plus compréhensible.
Par exemple:
```suggestion
        raise ValueError(f"value is lower than {self.threshold}")
        raise ValueError(f"value must be greater than {self.threshold}")
```